tests/stdlib/thttpclient_ssl.nim

```diff
@@ -124,4 +124,8 @@ when not defined(windows):
         let msg = getCurrentExceptionMsg()
         log "client: exception: " & msg
         # SSL_shutdown:shutdown while in init
-        check(msg.contains("shutdown while in init") or msg.contains("alert number 48"))
+        if not (msg.contains("shutdown while in init") or msg.contains("alert number 48") or
+          msg.contains("routines:CONNECT_CR_CERT:certificate verify failed")):
+          # pending https://github.com/nim-lang/Nim/pull/10558 simplify as: check(condition, msg)
```

Comment:
Why this comment? #10558 has been rejected.

Reply:
I intend to re-open and rebase this PR in the near future, I keep running into cases that calls for this feature, which is available with `doAssert cond, msg`

The rationale that was given for rejection `Tests and suites already have names.` does not take this kind of situation into account, where the context belongs in failure msg, not test suite tile, like here and so many other cases. Had the feature been avalable, you'd have been able to tell what went wrong without needing to modify the code as above and test locally.

Reply:
> I intend to re-open and rebase this PR in the near future

Ok, but this PR (or Nim codebase in general) shouldn't be your personal to-do list.